addons/auth/package.json

```diff
@@ -16,8 +16,10 @@
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint .",
     "lint:js:fix": "eslint . --fix",
+    "format": "yarn run format:js",
+    "format:js": "prettier --write '{addon,app,config,stories,tests}/**/*.js' *.js",
```

Comment:
I run the formatter and shows me 2 file changes, could we run formatter and add the changes to the PR?